content/blog/2019/08/2019-08-21-introduce-react-boilerplate-for-jenkins-plugin.adoc

```diff
@@ -0,0 +1,195 @@
+---
+layout: post
+title: "Introduce React Boilerplate for Jenkins Plugin"
+tags:
+- react
+- plugins
+- boilerplate
+- gsoc
+- gsoc2019
+author: jackshen
+---
+
+This boilerplate is part of the project link:https://summerofcode.withgoogle.com/projects/#6112735123734528[Working Hours UI Improvement] during
+link:https://summerofcode.withgoogle.com/[Google Summer of Code 2019], which improve the UI of Working Hours Plugin using this pattern to develop Jenkins plugin with React. The main repository could be found at link:https://github.com/jenkinsci/working-hours-plugin[Working Hours Plugin].
+
+
+## Overview
+
+Developing plugin for Jenkins has always been an easy way to do with its Jelly based UI render system, but Jelly seems to be pretty heavy when we need to use some more modern tools like React and if we need to make the plugin UI much customized, this is what this boilerplate is built for.
+
```

Comment:
It would be nice to have additional paragraph which summarizes the usage scenario. "With React Boilerplate, Jenkins plugin developers can build plugins with React just by... " or something like that